components/style/properties/longhand/svg.mako.rs

```diff
@@ -158,3 +158,27 @@ ${helpers.predefined_type("mask-image", "ImageLayer",
     animation_value_type="discrete",
     flags="CREATES_STACKING_CONTEXT",
     has_uncacheable_values="True" if product == "gecko" else "False")}
+
+${helpers.predefined_type(
```

Comment:
Yes they are, according the SVG2 spec every SVG presentation attribute is a css property. For example, you can do this in Chrome.